config/initializers/audited_hacks.rb

```diff
@@ -19,9 +19,40 @@ module AuditHacks
     alias_attribute :association_id, :associated_id
     alias_attribute :association_type, :associated_type
 
+    after_commit :log_to_stdout, on: :create, if: :logging_to_stdout?
+
     def self.delete_old
       where('created_at < ?', TTL.ago).delete_all
     end
+
+    def self.logging_to_stdout?
+      Features::LoggingConfig.config.audits_to_stdout
+    end
+
+    delegate :logging_to_stdout?, to: :class
+
+    def log_to_stdout
+      Rails.logger.info log_trail
+    end
+
+    protected
+
+    def log_trail
+      "[#{created_at}] [audit] #{kind} #{action} (#{to_json_safe})"
```

Comment:
I would double tag it with kind and action and following should be a JSON result so it can be machine readable, or even a context https://github.com/timberio/timber-ruby (but the license is unsure :/)

logger.tagged("audit", kind, action){ logger.info({created_at: created_at, context: to_h_safe}.to_json) }